### PR TITLE
Fix MC-266334

### PR DESCRIPTION
**Pull request by PadowYT2**
The bug was introduced in 1.20.2+ and was fixed in 1.21

### CODE REVIEW

File: patches/server/0095-Fix-MC-266334.patch [hunk 1]
```diff
@@ -0,0 +1,27 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: PadowYT2 <me@padow.ru>
+Date: Sun, 21 Jul 2024 15:16:54 +0300
+Subject: [PATCH] Fix MC-266334
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
```
Comment (PadowYT2) [inline patches/server/0095-Fix-MC-266334.patch:7]: Not sure if this should be added, but this was done by git

Comment (Dreeam-qwq) [reply patches/server/0095-Fix-MC-266334.patch:7]: Yeah, this is auto generated by git for compatibility

**Comment by HaHaWTH** [comment]
I prefer add a configuration for this fix.
Some servers may use this "bug" to build farms i think

**Comment by PadowYT2** [comment]
> I prefer add a configuration for this fix. Some servers may use this "bug" to build farms i think

This is rather kinda needed, when people build shulker farms, they need the armor stand to **not** break. That's why it was fixed in 1.21 and worked fine in 1.20.1

There are a lot of farms built for 1.20.1 and they don't work on 1.20.2+

**Comment by F3F5** [comment]
឵឵឵ ឵឵឵

**Comment by Dreeam-qwq** [comment]
Thanks!

**Pull request merged**